### DISCUSSION TITLE
Convert XML to Python in your browser

### DISCUSSION THREAD

**Discussion by alxbilger** [sofa-framework/sofa#5071] [Announcement]
Here is a convenient tool to convert an XML-based simulation scene to Python. It runs in a browser. No data is shared. Everything is local.

🔗 🔗 🔗 
https://alxbilger.github.io/sofa-xml_to_python/
🔗 🔗 🔗 

If you encounter a problem, don't hesitate to post an issue: https://github.com/alxbilger/sofa-xml_to_python/issues

**Comment by damienmarchal** [comment]
Nice.
